src/main/steps/applicant/support-you-need-during-case/SupportYouNeedDuringCaseController.test.ts

```diff
@@ -213,4 +213,27 @@ describe('SupportYouNeedDuringYourCaseController', () => {
     await controller.post(req, res);
     expect(req.session.userCase.applicantsFL401.response.keepDetailsPrivate).toEqual(undefined);
   });
+
+  test('Should update the SupportYouNeedDuringYourCase details if user id doesnot matches with respondent for DA', async () => {
+    req.session.user.id = '0c09b130-2eba-4ca8-a910-1f001bac01e7';
```

Comment:
is this a dummy user id? if yes, please ignore this comment and mark it as resolved.

Reply:
Yes it is a mock user ID.